src/main/java/seedu/badMaths/Parser.java

```
@@ -25,11 +25,11 @@ public String getFunction() {
         }
```

Should throw an exception here as the userInput is not valid.